src/main/java/mate.academy/service/FileReaderService.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 
+@Component
```

Comment:
you should use @Component annotation only over implementations of interfaces, not over interfaces